config/initializers/assets.rb

```diff
@@ -1,4 +1,3 @@
 Rails.application.config.assets.version = '1.0'
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
 Rails.application.config.assets.precompile += %w(application.css application.js bootstrap.min.js popper.js)
-# Rails.application.config.assets.unknown_asset_fallback = true
+Rails.application.config.assets.unknown_asset_fallback = true
```

Comment:
Is it ok for us to have this fallback enabled?

Reply:
true is the default value. So both lines are equivalent. Setting it to true is just more direct.